interfaces/PA-App/src/app/services/padata.service.ts

```diff
@@ -87,28 +87,20 @@ export class PaDataService {
       return this.featureNotAvailable();
     }
 
-    return this.paAccountApi.createAccount(username, password).then(
-      () => {
-        this.store('isLoggedIn', true, true);
-        this.authenticationStateSource.next(true);
-      }
-    );
+    return this.paAccountApi.createAccount(username, password);
   }
 
   async login(username: string, password: string): Promise<any> {
     if (this.useLocalStorage) {
       return this.featureNotAvailable();
     }
 
-    return this.paAccountApi.login(username, password).then(
-      () => {
-        this.store('isLoggedIn', true, true);
-        this.authenticationStateSource.next(true);
-      },
-      (error) => {
-        console.log(error);
-      }
-    );
+    return this.paAccountApi.login(username, password);
+  }
+
+  async setLoggedIn(): Promise<void> {
```

Comment:
This can be called from inside this class; As no-one else can "make you logged in" and want to 'set-this-user-as-logged-in'.